modules/cli-logging-in-web.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_cli/getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="cli-logging-in-web_{context}"]
+= Logging in to the OpenShift CLI using a web browser
+
+You can log in to the OpenShift CLI (`oc`) with the help of a web browser to access and manage your cluster. This allows users to avoid inserting their access token into the command line.
+
+== Logging in through a web browser with the CLI
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster.
+* You must have installed the OpenShift CLI (`oc`).
+* You must have a browser installed.
+
+.Procedure
+
+. Enter the `oc login` command with the '--web' flag:
++
+[source,terminal]
+----
+$ oc login <https://api.your-openshift-server.com> --web
```

Comment:
Replaceable values should be in the format of `<my_value>`. I'd update this to be something like `<cluster_url>`

```suggestion
$ oc login <cluster_url> --web
```

or an example value without the angle brackets, like:

```suggestion
$ oc login https://api.your-openshift-server.com --web
```